openeo/rest/auth/testing.py

```diff
@@ -169,15 +186,21 @@ def device_code_callback(
                 assert "code_challenge" not in params
             else:
                 raise ValueError(expect_code_challenge)
-        return json.dumps(
-            {
+
+        token = {
```

Comment:
FYI: this is not a (user) token, but this is verification data for the OIDC client (not the user)

Reply:
Thanks for clarifying that.